framework/PostProcessors/ComparisonStatisticsModule.py

```diff
@@ -36,6 +36,12 @@
 import Distributions
 #Internal Modules End--------------------------------------------------------------------------------
 
+# global number of integration points
+## note on Falcon that using 1e5 causes a seg fault.
+integrationSegments = int(1e5)
+#integrationSegments = int(5e4)
```

Comment:
remove the commented line

Reply:
These can be removed, I think.  I'll do that.